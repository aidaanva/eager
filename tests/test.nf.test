nextflow_pipeline {

    name "Test pipeline: NFCORE_EAGER"
    script "main.nf"
    tag "pipeline"
    tag "nfcore_eager"
    tag "test"

    test("test_profile") {

        when {
            params {
                outdir = "$outputDir"
            }
        }

        then {
            assertAll(
                { assert workflow.success },
                { assert snapshot(
                        path("$outputDir/"),
                    ).match()
                },
                { assert new File("$outputDir/pipeline_info/nf_core_pipeline_software_mqc_versions.yml").exists() },
                { assert new File("$outputDir/multiqc/multiqc_report.html").exists() },
            )
        }
    }
}
