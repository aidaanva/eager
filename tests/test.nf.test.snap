{
    "test_profile": {
        "content": [
            [
                [
                    "JK2782_JK2782_TGGCCGATCAACGA_BAM_Mammoth_MT_Krause_filtered.bam.bai:md5,d1823ee5a37ad32007d0923731127935",
                    "JK2782_JK2782_TGGCCGATCAACGA_BAM_Mammoth_MT_Krause_filtered.flagstat:md5,39fd430cc77fc71f71d75a221c1b7880",
                    "JK2782_JK2782_TGGCCGATCAACGA_BAM_Mammoth_MT_Krause_lengthonly.filtered.bam.bai:md5,159e97ddc9270c8d941f42f5553cf4fe",
                    "JK2782_JK2782_TGGCCGATCAACGA_Mammoth_MT_Krause_filtered.bam.bai:md5,49d2dbc1db9be5b66182a2a74f8793ab",
                    "JK2782_JK2782_TGGCCGATCAACGA_Mammoth_MT_Krause_filtered.flagstat:md5,a9fbd8c9a6f8bb02b191f4140f91d04a",
                    "JK2782_JK2782_TGGCCGATCAACGA_Mammoth_MT_Krause_lengthonly.filtered.bam.bai:md5,aa4c28b7c84d263fb864c783a58dc2b6",
                    "JK2802_JK2802_AGAATAACCTACCA_Mammoth_MT_Krause_filtered.bam.bai:md5,4b95a50ddb735e93566ddbe5bace461e",
                    "JK2802_JK2802_AGAATAACCTACCA_Mammoth_MT_Krause_filtered.flagstat:md5,f6c739b729c06e17ca205c64478ad823",
                    "JK2802_JK2802_AGAATAACCTACCA_Mammoth_MT_Krause_lengthonly.filtered.bam.bai:md5,64704942c7c21171ac0f4aade5e60069"
                ],
                [
                    "JK2782_JK2782_TGGCCGATCAACGA_BAM_8_Mammoth_MT_Krause.flagstat:md5,9a10991f5670a52975a622a8b056e2bc"
                ],
                [
                    [
                        [
                            "3pGtoA_freq.txt:md5,c32ef382b55fed96816df63993c520eb",
                            "3p_freq_misincorporations.txt:md5,2d4d9e2906c2238d27fa7e441bba2b33",
                            "5pCtoT_freq.txt:md5,d6e1401fb49387f83d842977ee47a9d2",
                            "5p_freq_misincorporations.txt:md5,af2192b8a97699519d060baa3c8982fe",
                            "DNA_comp_genome.txt:md5,0f27b55f4a7c2731f35761be4feddbfa",
                            "DNA_composition_sample.txt:md5,1336769b56151d9de542610c4a8ef662",
                            "DamagePlot.pdf:md5,7fa541ac9748a18366638efa30fd7d2a",
                            "DamagePlot_five_prime.svg:md5,089254de9c9677c9e95b0545962b18ee",
                            "DamagePlot_three_prime.svg:md5,73c201e2b08181a2be7a9e4ab462f752",
                            "DamageProfiler.log:md5,6f2a88aa36a26c509ce0cf9770f95f6f",
                            "Length_plot.pdf:md5,0ba43daa155d956e13f2cf665be14c3e",
                            "Length_plot_combined_data.svg:md5,586b7845b16599b369141ebaa21feb7d",
                            "Length_plot_forward_reverse_separated.svg:md5,f6b061f2dc16b712ce91d5255bd26837",
                            "dmgprof.json:md5,e829fb37f3e12842a6139e1d4cfe19dc",
                            "editDistance.txt:md5,7c00cd7cc4bd87c2da4debaa2704a66a",
                            "edit_distance.pdf:md5,84db19b30e4fccce00f5ef5c3a7f83c8",
                            "edit_distance.svg:md5,9991008d14b33c549b9f0c84a801ced9",
                            "lgdistribution.txt:md5,3c176293ea8e58d1d56a95e693727f7a",
                            "misincorporation.txt:md5,e25101ff61a0a3c377f8bc0252ee4451"
                        ],
                        [
                            "3pGtoA_freq.txt:md5,59aa1691a1c420dffeff4edc4a9cb0dd",
                            "3p_freq_misincorporations.txt:md5,15f64d35fc97bab2fed172fec292c3c9",
                            "5pCtoT_freq.txt:md5,51511bb6d9278afeb35099ab8b117119",
                            "5p_freq_misincorporations.txt:md5,fc0939e3e176deef644c0bd24cb0d141",
                            "DNA_comp_genome.txt:md5,76762da07599aff410c2ec531cd19720",
                            "DNA_composition_sample.txt:md5,0a4d26c613bbc4541eb51ec1344d6ef4",
                            "DamagePlot.pdf:md5,65a5556ca68357ae8cf2413379b376e5",
                            "DamagePlot_five_prime.svg:md5,545e5590aba1a3e093f064c62d092820",
                            "DamagePlot_three_prime.svg:md5,9e607d155b8de129a9014ddb633c53eb",
                            "DamageProfiler.log:md5,71d6f912bb8810221179db5175fae58f",
                            "Length_plot.pdf:md5,06fe56a624a7f0d774a6734791e40750",
                            "Length_plot_combined_data.svg:md5,a16b62d74c645d52f80a57b4556896a2",
                            "Length_plot_forward_reverse_separated.svg:md5,8d1bc4a2580d92ef52416131f3a1a044",
                            "dmgprof.json:md5,49a1a9b445abf48904ead652a16fb7e4",
                            "editDistance.txt:md5,5feeaa4b7018320fd9937e271299375c",
                            "edit_distance.pdf:md5,d7c2b09c275e2e08e66ec25048752c45",
                            "edit_distance.svg:md5,279b28d8d4f21ed8939514987c93156f",
                            "lgdistribution.txt:md5,3728eba42dd0b577a8566c103c342eb4",
                            "misincorporation.txt:md5,a2d9fed1148fe02e372f0905c7e15b40"
                        ],
                        [
                            "3pGtoA_freq.txt:md5,24ae86cb4dd4e558412344a98d582bae",
                            "3p_freq_misincorporations.txt:md5,fe312a520a01f198bf248f396b179aff",
                            "5pCtoT_freq.txt:md5,5a04f777d38930e205213298e70e09f1",
                            "5p_freq_misincorporations.txt:md5,8ecb71fca75a2f86cf44eb0cf4bfd269",
                            "DNA_comp_genome.txt:md5,b78d2826f0e1be4990120105c83d0b06",
                            "DNA_composition_sample.txt:md5,dd78c33e4d9367759e82c31247685a3b",
                            "DamagePlot.pdf:md5,6216e2251c6987366b36fd2c43ff007b",
                            "DamagePlot_five_prime.svg:md5,88f2661608e16519eabbc08643e96623",
                            "DamagePlot_three_prime.svg:md5,9e4e34f51734287b7892aa0971193481",
                            "DamageProfiler.log:md5,fffa551c97a1a5a8ff538d6d9de0c1ab",
                            "Length_plot.pdf:md5,af21790add82c31b74ac31a20bd1a005",
                            "Length_plot_combined_data.svg:md5,bb74a134ba7f37a21052b104a0202cc4",
                            "Length_plot_forward_reverse_separated.svg:md5,7cc6c37723cec43dc12cbd966e85dc80",
                            "dmgprof.json:md5,98c584c20659b11692cbd1e280deeb97",
                            "editDistance.txt:md5,059c352bf858c5ddb27e88253654e353",
                            "edit_distance.pdf:md5,715700625bd2cde1e7b9e68a7db31c9f",
                            "edit_distance.svg:md5,4c76fec3c327459066eba4ad5a4c88dc",
                            "lgdistribution.txt:md5,5e8760bf271b743aa3c7c96414fb9991",
                            "misincorporation.txt:md5,983d8fab464d67481b222aff57b06587"
                        ],
                        "versions.yml:md5,2814e619e5090776204ba0c2edfa07c2"
                    ]
                ],
                [
                    "JK2782_JK2782_TGGCCGATCAACGA_BAM_Mammoth_MT_Krause_dedupped.bam:md5,cafd8aec87c34a34a599aaf881bac4c2",
                    "JK2782_JK2782_TGGCCGATCAACGA_BAM_Mammoth_MT_Krause_dedupped.bam.bai:md5,196d7c4df7b32afc5209fdf7fff96a37",
                    "JK2782_JK2782_TGGCCGATCAACGA_BAM_Mammoth_MT_Krause_dedupped.flagstat:md5,715d840d3bab5a05934825cd9b799ef3",
                    "JK2782_JK2782_TGGCCGATCAACGA_Mammoth_MT_Krause_dedupped.bam:md5,9268839800197f8f98c9544f0d5b0847",
                    "JK2782_JK2782_TGGCCGATCAACGA_Mammoth_MT_Krause_dedupped.bam.bai:md5,422c90355dfa3149062dc4c2618b6078",
                    "JK2782_JK2782_TGGCCGATCAACGA_Mammoth_MT_Krause_dedupped.flagstat:md5,dde5f59b006afdae2ff5fab85bc8ea92",
                    "JK2802_JK2802_AGAATAACCTACCA_Mammoth_MT_Krause_dedupped.bam:md5,ee214d5a80b67ab330cb29004d23c2ac",
                    "JK2802_JK2802_AGAATAACCTACCA_Mammoth_MT_Krause_dedupped.bam.bai:md5,68c53ce698704d94af86678b0b96293e",
                    "JK2802_JK2802_AGAATAACCTACCA_Mammoth_MT_Krause_dedupped.flagstat:md5,fb4d234136f242c20db27979caa604cf"
                ],
                [
                    "JK2782_Mammoth_MT_Krause.bam:md5,565b1cd3375fc0c013b8dd3ee74fa98e",
                    "JK2782_Mammoth_MT_Krause.bam.bai:md5,8008bf95ad184f82254a82cbb44cc872",
                    "JK2782_Mammoth_MT_Krause.flagstat:md5,04ff612d7bb381cca5bda876b7ed7d13",
                    "JK2802_Mammoth_MT_Krause.bam:md5,8ebe9c477dbba95ced830fbe09fecd1b",
                    "JK2802_Mammoth_MT_Krause.bam.bai:md5,541baf7e85b27e4c210373cd61f93d52",
                    "JK2802_Mammoth_MT_Krause.flagstat:md5,fb4d234136f242c20db27979caa604cf"
                ],
                [
                    "JK2782_JK2782_TGGCCGATCAACGA_Mammoth_MT_Krause.flagstat:md5,98a2f8bb2aa9678e4cf9578ee9ab3b0e",
                    "JK2782_JK2782_TGGCCGATCAACGA_Mammoth_MT_Krause_sorted.bam:md5,7fa16f8c23d535cfb4b66075be50a097",
                    "JK2782_JK2782_TGGCCGATCAACGA_Mammoth_MT_Krause_sorted.bam.bai:md5,6225a8b059bc4396a60165bd9d8dff91",
                    "JK2802_JK2802_AGAATAACCTACCA_Mammoth_MT_Krause.flagstat:md5,cb86b29c65c8c123a8863330316c6400",
                    "JK2802_JK2802_AGAATAACCTACCA_Mammoth_MT_Krause_sorted.bam:md5,da658a3f61f8969d1f5d3126c39c2bf7",
                    "JK2802_JK2802_AGAATAACCTACCA_Mammoth_MT_Krause_sorted.bam.bai:md5,e43c10077352dc65ecc39d76f421338f"
                ],
                [
                    [
                        "JK2782_JK2782_TGGCCGATCAACGA_BAM_Mammoth_MT_Krause_breadth.bed:md5,45b641735c8e2b4be10766a5c246068d",
                        "JK2782_JK2782_TGGCCGATCAACGA_BAM_Mammoth_MT_Krause_depth.bed:md5,2b6ffd36b2fb3b8b2b52b928d7a63158",
                        "JK2782_JK2782_TGGCCGATCAACGA_Mammoth_MT_Krause_breadth.bed:md5,5286794122bb3c67d49cc858d40f0c31",
                        "JK2782_JK2782_TGGCCGATCAACGA_Mammoth_MT_Krause_depth.bed:md5,5065bd5c0001f20bf4f03d62297cf4ca",
                        "JK2802_JK2802_AGAATAACCTACCA_Mammoth_MT_Krause_breadth.bed:md5,16b5f8952d87e236a63e681293691832",
                        "JK2802_JK2802_AGAATAACCTACCA_Mammoth_MT_Krause_depth.bed:md5,1b4f117a16c712b7c67809d7e4ec279f",
                        "versions.yml:md5,f6c22504bd960903cef378d15aa1cf99"
                    ],
                    [
                        "JK2782_JK2782_TGGCCGATCAACGA_BAM_Mammoth_MT_Krause_percent_on_target_mqc.json:md5,a3ccef0b8d19d69457b817898a2a5604",
                        "JK2782_JK2782_TGGCCGATCAACGA_Mammoth_MT_Krause_percent_on_target_mqc.json:md5,58df3f7e6ba65086daa94e49fe6b60f3",
                        "JK2802_JK2802_AGAATAACCTACCA_Mammoth_MT_Krause_percent_on_target_mqc.json:md5,f771d9d15a744ea8a7031d36551ec7b3"
                    ],
                    [
                        "JK2782_JK2782_TGGCCGATCAACGA_BAM_Mammoth_MT_Krause.c_curve.txt:md5,ba8059ca2396fb898593c1e66e56a7bc",
                        "JK2782_JK2782_TGGCCGATCAACGA_BAM_Mammoth_MT_Krause.command.log:md5,d41d8cd98f00b204e9800998ecf8427e",
                        "JK2782_JK2782_TGGCCGATCAACGA_Mammoth_MT_Krause.c_curve.txt:md5,817f35dc7ce2983a7d0f558debd6e94b",
                        "JK2782_JK2782_TGGCCGATCAACGA_Mammoth_MT_Krause.command.log:md5,d41d8cd98f00b204e9800998ecf8427e",
                        "JK2802_JK2802_AGAATAACCTACCA_Mammoth_MT_Krause.c_curve.txt:md5,1384a88b531ab872c0069e71fccbe97a",
                        "JK2802_JK2802_AGAATAACCTACCA_Mammoth_MT_Krause.command.log:md5,d41d8cd98f00b204e9800998ecf8427e",
                        "versions.yml:md5,04edb6451354c81b20ffba3102b625b8"
                    ],
                    [
                        [
                            [
                                [
                                    [
                                        [
                                            "agogo.css:md5,bd757b1a7ce6fdc0288ba148680f4583",
                                            "ajax-loader.gif:md5,ae6667053ad118020b8e68ccf307b519",
                                            "basic.css:md5,25b2823342c0604924a2870eeb4e7e94",
                                            "bgfooter.png:md5,ed01bb040346e4623cc87de331ddb4e1",
                                            "bgtop.png:md5,626a50532dc6e5adbdd968746ef318e8",
                                            "comment-bright.png:md5,0c850bb4920b581bf5e5dba5fa493a64",
                                            "comment-close.png:md5,2635dda49c823e8122d4d11ed385f33d",
                                            "comment.png:md5,882e40f3d6a16c6ed35659b105251525",
                                            "doctools.js:md5,5ff571aa60e63f69c1890283e240ff8d",
                                            "down-pressed.png:md5,ebe8979581eda700fb234a73c661a4b9",
                                            "down.png:md5,f6f3c819cc7ca27d7fd3347e5e7ffe0f",
                                            "file.png:md5,6587e59c55e626744eb6fc11129d99a7",
                                            "jquery.js:md5,10092eee563dec2dca82b77d2cf5a1ae",
                                            "minus.png:md5,8d572395aa95c89584a09813ada4dfa1",
                                            "plus.png:md5,0125e6faa04e2cf0141a2d599d3bb220",
                                            "pygments.css:md5,d625a0adb949f181bd0d3f1432b0fa7f",
                                            "qualimap_logo_small.png:md5,7526f145a97be4682fd59e27dda4f71b",
                                            "report.css:md5,7a5f09eaf7c176f966f4e8854168b812",
                                            "searchtools.js:md5,d550841adeedc8ed47c40ee607620937",
                                            "underscore.js:md5,db5ba047a66617d4cd3e8c5099cc51db",
                                            "up-pressed.png:md5,8ea9bd109342f87fee97943b479c6f7e",
                                            "up.png:md5,ecc373278454cc8ecc12d6ca69e55b36",
                                            "websupport.js:md5,9e61e1e8a7433c56bd7e5a615affcf85"
                                        ],
                                        "genome_results.txt:md5,d520d13c02f78d7f74bcefed4864ed61",
                                        [
                                            "genome_coverage_0to50_histogram.png:md5,80aa3c94722e4c938393a45b7015a813",
                                            "genome_coverage_across_reference.png:md5,64b7b05ec75f24d60cb779d234fc2399",
                                            "genome_coverage_histogram.png:md5,ef7314fa488b67773f9174cfd96a7b64",
                                            "genome_coverage_quotes.png:md5,7e83e24f741f74978d5c1ed479d86eb2",
                                            "genome_gc_content_per_window.png:md5,e0e20c8a65b6d41c88a6a2ef44a9ec1a",
                                            "genome_homopolymer_indels.png:md5,e2cdb644cfca349cc7a7151175d32f0d",
                                            "genome_mapping_quality_across_reference.png:md5,7108d78085bef09350214151e3429b89",
                                            "genome_mapping_quality_histogram.png:md5,a9f0ca725de66c600d127baeff3d63ea",
                                            "genome_reads_content_per_read_position.png:md5,650b98e61e3bc39351effe0a79c21876",
                                            "genome_uniq_read_starts_histogram.png:md5,d1e870f849ba228e52d5be513eb86c8a"
                                        ],
                                        "qualimapReport.html:md5,9e8d6d04964feefc18211a155308a144",
                                        [
                                            "coverage_across_reference.txt:md5,b724388bf0301fb73cc95adcd99f3e9f",
                                            "coverage_histogram.txt:md5,402146079b9e04bc1acc5f02e7b61d7c",
                                            "duplication_rate_histogram.txt:md5,e6f1a6e48ba04b2aa69a09b56e10962b",
                                            "genome_fraction_coverage.txt:md5,058852542e5052d456c3e41b2ad7fe68",
                                            "homopolymer_indels.txt:md5,2cc3cb5d9105a4d5feca2b3dbe32494e",
                                            "mapped_reads_gc-content_distribution.txt:md5,3262dd7abf0325ccc7935277ecff9458",
                                            "mapped_reads_nucleotide_content.txt:md5,714ea0b92f3d0919bda34ee521035b47",
                                            "mapping_quality_across_reference.txt:md5,bde42f64a3d9d8611a76a2402fa29fb2",
                                            "mapping_quality_histogram.txt:md5,3943155d987a6d966950b4b77666cd4e"
                                        ]
                                    ],
                                    "versions.yml:md5,1f32280851a9253616996ce3b72d0c04"
                                ]
                            ],
                            [
                                [
                                    [
                                        [
                                            "agogo.css:md5,bd757b1a7ce6fdc0288ba148680f4583",
                                            "ajax-loader.gif:md5,ae6667053ad118020b8e68ccf307b519",
                                            "basic.css:md5,25b2823342c0604924a2870eeb4e7e94",
                                            "bgfooter.png:md5,ed01bb040346e4623cc87de331ddb4e1",
                                            "bgtop.png:md5,626a50532dc6e5adbdd968746ef318e8",
                                            "comment-bright.png:md5,0c850bb4920b581bf5e5dba5fa493a64",
                                            "comment-close.png:md5,2635dda49c823e8122d4d11ed385f33d",
                                            "comment.png:md5,882e40f3d6a16c6ed35659b105251525",
                                            "doctools.js:md5,5ff571aa60e63f69c1890283e240ff8d",
                                            "down-pressed.png:md5,ebe8979581eda700fb234a73c661a4b9",
                                            "down.png:md5,f6f3c819cc7ca27d7fd3347e5e7ffe0f",
                                            "file.png:md5,6587e59c55e626744eb6fc11129d99a7",
                                            "jquery.js:md5,10092eee563dec2dca82b77d2cf5a1ae",
                                            "minus.png:md5,8d572395aa95c89584a09813ada4dfa1",
                                            "plus.png:md5,0125e6faa04e2cf0141a2d599d3bb220",
                                            "pygments.css:md5,d625a0adb949f181bd0d3f1432b0fa7f",
                                            "qualimap_logo_small.png:md5,7526f145a97be4682fd59e27dda4f71b",
                                            "report.css:md5,7a5f09eaf7c176f966f4e8854168b812",
                                            "searchtools.js:md5,d550841adeedc8ed47c40ee607620937",
                                            "underscore.js:md5,db5ba047a66617d4cd3e8c5099cc51db",
                                            "up-pressed.png:md5,8ea9bd109342f87fee97943b479c6f7e",
                                            "up.png:md5,ecc373278454cc8ecc12d6ca69e55b36",
                                            "websupport.js:md5,9e61e1e8a7433c56bd7e5a615affcf85"
                                        ],
                                        "genome_results.txt:md5,f244a1f31395a4c980ad3b39d525c231",
                                        [
                                            "genome_coverage_0to50_histogram.png:md5,405e3b012c5092a33e099042c7a479f8",
                                            "genome_coverage_across_reference.png:md5,96f145a33a8c56cdaebb7f0197c476e4",
                                            "genome_coverage_histogram.png:md5,53ad483690ce3db0286c493421b165f5",
                                            "genome_coverage_quotes.png:md5,dea5046f36d0963ee25ca1ac51154382",
                                            "genome_gc_content_per_window.png:md5,8206f9e95c9889911668f38a154faa52",
                                            "genome_homopolymer_indels.png:md5,66811ef0c7fe5e32392e8dfced3c68f1",
                                            "genome_mapping_quality_across_reference.png:md5,61a09c6116481dc68b408683d8d5d09c",
                                            "genome_mapping_quality_histogram.png:md5,50c3e22a2d17c314ad30b06ff7993396",
                                            "genome_reads_clipping_profile.png:md5,386954c65ba40aee3129aa3fd840445d",
                                            "genome_reads_content_per_read_position.png:md5,587bd7978b3e18a65099f06c119c3f04",
                                            "genome_uniq_read_starts_histogram.png:md5,4f7305df090a4ad3bc87342740d414ef"
                                        ],
                                        "qualimapReport.html:md5,ec14c3852748c6ef339d2c4cb424d1f2",
                                        [
                                            "coverage_across_reference.txt:md5,3be6bd7fef9e81d56792b58e902949ed",
                                            "coverage_histogram.txt:md5,8aaee99c160b7b5e030ee6fc891d9837",
                                            "duplication_rate_histogram.txt:md5,ffd92c4701b07a4f3b6a580928770fe4",
                                            "genome_fraction_coverage.txt:md5,f2e4bc958f9df721a6adccbc69de9706",
                                            "homopolymer_indels.txt:md5,e6beb36c6aab5de9cb6d9e2357f82995",
                                            "mapped_reads_clipping_profile.txt:md5,f3ee469d515a49d50ac4e438ddfc8b57",
                                            "mapped_reads_gc-content_distribution.txt:md5,818ed0f391e3d08fee4f102721127976",
                                            "mapped_reads_nucleotide_content.txt:md5,058c7dc070f76bc2c1913595070f9066",
                                            "mapping_quality_across_reference.txt:md5,fb2c0f2f4c95c698ab67520cb3f2db5a",
                                            "mapping_quality_histogram.txt:md5,4c5a2755f3eec182031390f7bb0d4d0f"
                                        ]
                                    ],
                                    "versions.yml:md5,1f32280851a9253616996ce3b72d0c04"
                                ]
                            ]
                        ]
                    ]
                ],
                [
                    [
                        "multiqc.log:md5,bc0d5556077e2fc9bf7c98c20164d991",
                        "multiqc_citations.txt:md5,24cca4fa791953701472e4d193ad290b",
                        "multiqc_damageprofiler_metrics.txt:md5,6ccd45ddc65b12421006990249ef199f",
                        "multiqc_data.json:md5,7585d901bed25cc9f3abdb110a09e216",
                        "multiqc_fastp.txt:md5,d58fc30991f81e6d3de99e34ad321aff",
                        "multiqc_fastqc.txt:md5,3355a9c1b391caecdad4ec6f5c4ea222",
                        "multiqc_general_stats.txt:md5,eab948a6baae5f8f98402c9a73f9741d",
                        "multiqc_qualimap_bamqc_genome_results.txt:md5,7fdeab4023681b3a73c981c8e14f2029",
                        "multiqc_samtools_flagstat.txt:md5,7b8a63ae77afe02e1ddec97f97a0a95d",
                        "multiqc_software_versions.txt:md5,a61716c40c5206158c3361ce023e0b73",
                        "multiqc_sources.txt:md5,35f2f74c321b2429072fd07361b586b8",
                        "preseq.txt:md5,b045237854edfec9bc858684917ffdac"
                    ],
                    [
                        
                    ],
                    "multiqc_report.html:md5,cfaca9e456db5fc74a2d094e070d487e"
                ],
                [
                    "execution_report_2024-04-26_10-51-33.html:md5,c48616609bc6de3602562a24005cb8bb",
                    "execution_timeline_2024-04-26_10-51-33.html:md5,be16319d60c049e304b989647d38490c",
                    "nf_core_pipeline_software_mqc_versions.yml:md5,0d210e80c3b9044d1371fa3aeb14cde7",
                    "params_2024-04-26_10-51-46.json:md5,126f9c285f89f191702a99eef05cd103",
                    "pipeline_dag_2024-04-26_10-51-33.html:md5,a70880047226f19b66406452acb868cf"
                ],
                [
                    "JK2782_JK2782_TGGCCGATCAACGA_L1.fastp.html:md5,3dbc5627920a6efb967eb1673b0788e8",
                    "JK2782_JK2782_TGGCCGATCAACGA_L1.fastp.json:md5,f9ac29dc38bd5e51beb9ea5a8a8e22f3",
                    "JK2782_JK2782_TGGCCGATCAACGA_L1.fastp.log:md5,e7e2e16afe04f4d70e5fad458079c64b",
                    "JK2802_JK2802_AGAATAACCTACCA_L2.fastp.html:md5,7991a49042595d54f1ce3d862a088948",
                    "JK2802_JK2802_AGAATAACCTACCA_L2.fastp.json:md5,582794e200f2ddf0aad317ab42f88e79",
                    "JK2802_JK2802_AGAATAACCTACCA_L2.fastp.log:md5,cc245eaecedc85ef9e4d0f0e6b1f1c45",
                    [
                        "null_fastqc.html:md5,70581cbcf992687d091d62bfb9ca0bde",
                        "null_fastqc.zip:md5,1cfab845cc9a6514801e7ad8b4f5d65a",
                        "versions.yml:md5,42fbb757df8018b4595df690155d41d2"
                    ],
                    [
                        "null_1_fastqc.html:md5,e3f59126623cfa335401f0167c0d5ee5",
                        "null_1_fastqc.zip:md5,6552a3b03d67bc98e4db97f3e621074f",
                        "null_2_fastqc.html:md5,ac5477d5a959efe94d0f4088344bedb9",
                        "null_2_fastqc.zip:md5,229b65a35e76ac8b720e6baa8f05ff32",
                        "null_fastqc.html:md5,843572a1e2cd975451f7e718d5c5893f",
                        "null_fastqc.zip:md5,ad5af298d03ffe37adb83e1818f9f634",
                        "versions.yml:md5,b4672add0ce3c990afa1a2bc9f6a7c87"
                    ]
                ],
                [
                    "JK2782_JK2782_TGGCCGATCAACGA_BAM.bam.bai:md5,159e97ddc9270c8d941f42f5553cf4fe"
                ]
            ]
        ],
        "meta": {
            "nf-test": "0.8.4",
            "nextflow": "24.03.0"
        },
        "timestamp": "2024-04-26T10:54:24.5666711"
    }
}